profile/README.md

```diff
@@ -1,36 +1,60 @@
+
+**What’s included in the Benchmark Dataset?**  
+- Publicly available, labeled field boundaries from around the world  
+- Uniform data formatting and metadata schema  
+- A maintained subset of training/validation/test data  
+- Tools to evaluate model performance on shared benchmarks  
+- Contribution guidelines for submitting new datasets
+
+
+### 🧠 FTW Baseline Models  
+Reference implementations of machine learning models trained on the FTW Benchmark Dataset. These models offer starting points and performance baselines for researchers and practitioners.
 
-1. **Near-Global Coverage**: FTW spans four continents—Europe, Africa, Asia, and South America—covering diverse agricultural landscapes across 24 countries. This extensive geographic coverage allows for the development of models that can generalize well to different agricultural practices and field types.
+### 🛠️ FTW Tools  
+A growing suite of tools to support end-to-end workflows:
+- **Data access tools**: Python libraries and CLI tools to download and work with the benchmark dataset  
+- **Inference tools**: Scripts and utilities to run field boundary detection on new imagery  
+- **Formatting tools**: Resources to convert raw data into FTW-compliant format (e.g., fiboa to FTW train/test split)  
+- **QGIS Plugin**: A plugin to browse and interact with field data within QGIS
 
-2. **Large-Scale Dataset**: With approximately 1.6 million parcel boundaries and over 70,000 samples, FTW is significantly larger than previously available datasets. Each sample includes instance and semantic segmentation masks paired with multi-date, multi-spectral Sentinel-2 satellite images, enabling detailed temporal and spectral analysis.
+### 🌐 FTW Web App  
+A browser-based interface to explore, select, and analyze scenes using FTW models and data. 
 
-3. **Multi-class Segmentation**: The dataset provides masks for both instance segmentation and semantic segmentation with different classes, including:
-   - **Instance Segmentation Masks**: To identify individual fields.
-   - **Semantic Segmentation Masks**: 
-     - Two-class masks: Background and polygon (field).
-     - Three-class masks: Background, polygon (field), and boundaries.
+### 📡 FTW Source Data  
+Curated public datasets of manually labeled field boundaries—such as those hosted on [fiboa.org](https://fiboa.org)—that may serve as inputs to the benchmark dataset. These are raw sources prior to FTW formatting.
 
-4. **Spectral Richness**: The dataset includes RGB (Red, Green, Blue) and NIR (Near-Infrared) spectral bands from Sentinel-2 images.
+### 🗺️ FTW Data Products  
+Large-scale outputs of model-inferred field boundaries (e.g., country-wide field maps). These are versioned and published for external research, planning, and applications.
```

Comment:
```suggestion
Large-scale outputs of model-inferred field boundaries (e.g., country-wide field maps). These are versioned and published for external evaluation, planning, and applications.
```

I wouldn't emphasize research so much here. Also should explain here that none are produced yet, but that there are in progress efforts for Latin America, Kenya, Vietnam, etc. (I forget the exact list, but we should put it). And that when the model quality is high enough we'll aim to produce a global-scale product.